### DISCUSSION TITLE
Upgrading Quasar v1 project (to Quasar v2, Vue 3, Typescript & Vite)

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#14076] [General - Components / Directives / etc] [answered]
Hello,

we would like to update our Quasar v1 / Vue 2 project as follows: 

- use Typescript
- use Vue 3
- use Quasar v2
- use Vite

Based on your experience, what steps would you recommend taking, e.g. what should be done first, second, etc.?

Thanks all!

**Comment by MilosPaunovic** [comment, marked as answer]
Recommendation is to follow official upgrade guide:
https://quasar.dev/start/upgrade-guide

**Comment by i-mage** [comment]
Thank you!
